pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -306,7 +306,7 @@ func (r *ScaleREST) Destroy() {
 func (r *ScaleREST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
 	obj, err := r.store.Get(ctx, name, options)
 	if err != nil {
-		return nil, errors.NewNotFound(apps.Resource("deployments/scale"), name)
+		return nil, err
```

Comment:
I _think_ this is fine because we generally don't consider these strings to be an API but maybe we should call it out in a release note?